data/hai/hai reveal 5 philosophers.txt

```diff
@@ -631,124 +493,98 @@ fleet "Smuggler (Hai Traffickers)"
 	names "civilian"
 	cargo 0
 	personality timid
-	variant 3
+	variant 10
```

Comment:
Why so many blackbirds?

Reply:
Not anymore. I just rebalanced the variant weights.